### PR TITLE
Addsupportfor kafka

### PR DESCRIPTION
**Pull request by fbalicchia**
Hi,
Add support for Kafka with client 0.9.0.1. 
At the moment I'm trying to  determine to add unittest the with kafka-unit or simple extending TestRule interface, but I hope that is not blocker for review the commit.
On the other side before commit ad I've try it with filebeat and It works for me.
Thanks in advance for help and if you have suggest are wellcome

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.graylog.plugins</groupId>
         <artifactId>graylog-plugin-parent</artifactId>
-        <version>2.3.0-alpha.4-SNAPSHOT</version>
+        <version>2.2.3</version>
```
Comment (joschi) [inline pom.xml]: Please revert this change.

File: pom.xml [hunk 2]
```diff
@@ -73,7 +73,8 @@
         <maven.install.skip>true</maven.install.skip>
         <maven.deploy.skip>true</maven.deploy.skip>
         <maven.site.skip>true</maven.site.skip>
-        <graylog.version>2.3.0-alpha.4-SNAPSHOT</graylog.version>
+        <graylog.version>2.2.3</graylog.version>
```
Comment (joschi) [inline pom.xml]: Please revert that change.

File: src/main/java/org/graylog/plugins/beats/input/BeatKafkaInput.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import org.graylog2.plugin.configuration.Configuration;
+import org.graylog2.plugin.inputs.MessageInput;
+import org.graylog2.plugin.inputs.annotations.ConfigClass;
+import org.graylog2.plugin.inputs.annotations.FactoryClass;
+
+import javax.inject.Inject;
+
+/**
+ * Created by fbalicchia on 10/06/17.
+ */
```
Comment (joschi) [inline src/main/java/org/graylog/plugins/beats/input/BeatKafkaInput.java]: Please remove this Javadoc comment.

**Review by joschi — CHANGES_REQUESTED** [review]

**Comment by fbalicchia** [comment]
Hi,
javaDoc is removed and revert reference to old graylog version. I've remove ssl configuration cause at the moment is not ready tobe review

**Comment by salekseev** [comment]
@joschi please take a look at this PR one more time. I could really make use this code. Thanks.

**Comment by kroepke** [comment]
Hi @fbalicchia! Sorry for the long delay.

We are in the process of moving the beats plugin into the [Graylog server](https://github.com/Graylog2/graylog2-server) repository because it is widely used and is being shipped by default now anyway.
As this PR is currently not in a state to be merged automatically, I'd like to ask you to reopen it against the server repository once the beats plugin has been merged over.
I'll close this one now.

Thanks!

**Comment by kroepke** [comment]
For when that has happened, please watch https://github.com/Graylog2/graylog-plugin-beats/issues/28

**Pull request closed without merging**